package.json

```diff
@@ -33,10 +36,16 @@
   },
   "homepage": "https://github.com/gagern/node-libtidy#readme",
   "devDependencies": {
+    "@types/chai": "^3.5.0",
+    "@types/mocha": "^2.2.40",
+    "@types/node": "^7.0.12",
```

Comment:
I was mostly referring to those which have unscoped counterparts. No need to change the set of files, just the versions.